### DISCUSSION TITLE
File Provisioner for Infrastructure provider Docker

### DISCUSSION THREAD

**Discussion by Sir-gits-a-lot** [kubernetes-sigs/cluster-api#8884] [Q&A]
Is there an option in CAPI to provision files to the nodes of the Workload cluster after they are provisioned? Kind of like what we have in Terraform where we can use null resource with ssh and file provisioner to have some files placed inside the nodes once they are launched by Terraform. 
I am trying to find some equivalent in CAPI. If anybody knows, please let me know.

**Comment by killianmuldoon** [comment]
/close 

Please use issues for questions like this or ask in the cluster-api slack channel.
